lib/ratchet/plug/data.ex

```diff
@@ -1,6 +1,10 @@
 defmodule Ratchet.Plug.Data do
+  @callback data(Plug.Conn.t) :: Plug.Conn.t
```

Comment:
data/1 doesn't _have_ to return a Plug.Conn, it could probably be anything. It's the "value" and key/value pair being plugged into the data structure. Recommending changing this to `any`.